app/build.gradle

```diff
@@ -119,9 +120,11 @@ dependencies {
     // Android support libraries
     implementation "androidx.core:core:$androidx_core_version"
     implementation "androidx.recyclerview:recyclerview:$androidx_recycler_version"
-    implementation "androidx.legacy:legacy-support-v4:$legacy_support"
+    implementation "androidx.legacy:legacy-support-v4:$legacy_support_v4"
+    implementation "androidx.appcompat:appcompat:$appcompat_appcompat"
     implementation "com.google.android.material:material:$androidx_material_version"
-    implementation "com.android.support.constraint:constraint-layout:$constraint_layout"
+    implementation "androidx.constraintlayout:constraintlayout:$constraintlayout"
+    implementation "com.android.support.constraint:constraint-layout:$constraintlayout"
```

Comment:
If you're using androidx import for constraintlayout, you could remove com.android dependency